src/styles/_catalog.scss

```diff
@@ -1,22 +1,32 @@
+$card-width: 200px;
+$card-gap: 48px;
```

Comment:
it's better to move it to the `variables` file

Reply:
Yeah, I've got a question about that. If we have even one or two variables, should we make another file for variables?